R/rpubs.R

```diff
@@ -90,7 +90,8 @@ rpubsUpload <- function(title,
 
     # create the tarball
     tarfile <- tempfile("package", fileext = ".tar.gz")
-    utils::tar(tarfile, files = ".", compression = "gzip", tar = "internal")
+    tarImplementation <- Sys.getenv("RSCONNECT_TAR", "internal")
+    utils::tar(bundlePath, files = ".", compression = "gzip", tar = tarImplementation)
```

Comment:
Do you think it would be worth creating a helper function here (just to share code)?

Reply:
Sure. I didn't do that initially because the verbosity options were not consistent.

Reply:
Done, and tested publishing to Connect, shinyapps.io, and rpubs.